app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,40 +1,31 @@
 import React, { useState } from "react";
 
-import { Alert } from "neetoui";
+import { Alert, Toastr } from "neetoui";
 
-import notesApi from "apis/notes";
-
-const DeleteAlert = ({
-  refetch,
-  onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
-}) => {
-  const [deleting, setDeleting] = useState(false);
+const DeleteAlert = onClose => {
+  const [open, setOpen] = useState(true);
 
   const handleDelete = async () => {
     try {
-      setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
-      onClose();
-      setSelectedNoteIds([]);
-      refetch();
+      closeAlert();
+      Toastr.success("Note has been successfully deleted.");
     } catch (error) {
       logger.error(error);
-      setDeleting(false);
     }
   };
 
+  const closeAlert = () => {
+    setOpen(false);
+    onClose;
```

Comment:
What's the point of writing this function here without invoking it?

Reply:
Done, Thanks.